File: program/find-cube-root-of-a-number/find_cube_root_of_a_number.dart
import 'dart:io';
import 'dart:math';
void main(){
  print("Enter a number :");
  var number =int.parse(stdin.readLineSync());
  var cube_root = pow(number,1/3).round();
  print('Cube root of the given number : $cube_root');
}

File: program/find-cube-root-of-a-number/findCubeRootOfANumber.scala
import scala.io.StdIn._;

object cubeRoot{  
    def main(args:Array[String]):Unit={  
        var num:Double    = 0;
        var res:Double = 0;
           
        print("Enter a number: ")
        num= readDouble()

        res= scala.math.cbrt(num)
        println("\nCube root of the given number : "+res);
    }  
}
